daml-foundations/daml-ghc/daml-prim-src/GHC/Num.daml

```diff
@@ -132,11 +133,12 @@ instance Signed Decimal where
     signum x = if x == 0.0 then 0.0 else if x <= 0.0 then (-1.0) else 1.0
     abs x = if x <= 0.0 then negate x else x
 
+-- | The `Fractional` class.
```

Comment:
We should expand to say this is for types that can be divided. `recip x` does not need to be defined for all `x : a`, for instance it is not defined for `0.0 : Decimal`. There are also some laws:
- when `recip x` is defined, it need to be the inverse of `x` wrt `*`, ie. `x * recip` = `munit` (in fact, our `Decimal` types only obeys this law only up to some precision)
- when `recip y` is defined, then `x / y` = `x * recip y`

Reply:
Added!